handlers/edit_teacher.go

```diff
@@ -0,0 +1,29 @@
+package handlers
+
+import (
+	runtime "github.com/ahab94/ApnaSchool"
+	"github.com/ahab94/ApnaSchool/gen/restapi/operations"
+	"github.com/go-openapi/runtime/middleware"
+)
+
+// NewEditTeacher handles a request for editing a teacher
+func NewEditTeacher(rt *runtime.Runtime) operations.EditTeacherHandler {
+	return &editTeacher{rt: rt}
+}
+
+type editTeacher struct {
+	rt *runtime.Runtime
+}
+
+// Handle the put teacher request
+func (d *editTeacher) Handle(params operations.EditTeacherParams) middleware.Responder {
+	val, err := d.rt.Service().RetrieveTeacher(params.ID)
```

Comment:
'Val' is not a good variable name. It should be teacher' representative of the object.

Reply:
sorry, i added this one in a hurry just for testing purpose